### PR TITLE
[WIP]First Cut design document for Orchestration and Automation.

### PR DESCRIPTION
**Pull request by kumarashit**
Design doc for orchestration  and automation

### CODE REVIEW

File: specs/capri/orchestration_automation_design.md [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+It is the external framework to organize, centralize, and standardize workflows. 
+
+ Workflow
+It stitches actions together, define the order, transition conditions, and passing the data. 
+
+ Actions
+Actions are pieces of code that can perform arbitrary automation tasks.
+
+## Design details
+![Orchestration Design Diagram](Orchestration_and_Automation_Design.PNG?raw=true "Orchestration Design Diagram")
```
Comment (xing-yang) [inline specs/capri/orchestration_automation_design.md]: Something is not right.  I cannot view this diagram when I view the doc.

Comment (himanshuvar) [inline specs/capri/orchestration_automation_design.md]: @kumarashit Please change the extension to .png

Comment (himanshuvar) [reply specs/capri/orchestration_automation_design.md]: @xing-yang file extension needs change to .png
We will update the design docs for other contents like API, sequence diagram.

**Comment by xing-yang** [comment]
https://github.com/opensds/design-specs/pull/35 is merged.  Closing this one.

**Pull request closed without merging**